src/reducers/sample.js

```diff
@@ -0,0 +1,18 @@
+import { ADD_DATA } from '../actions/actionTypes';
+import initialState from '../store/initialState';
+
+/**
+ * This is a sample reducer function
+ *
+ * @param {object} [state=initialState] - Initial state to load
+ * @param {Function} action - Action to dispatch
+ * @returns {object}
+ */
+export default function rootReducer(state = initialState, action) {
```

Comment:
Are you using this file? If yes, please I'll like to know what it does as I don't really know what it does.

Reply:
resolved